src/Solver/time_stepping/GhostTimeClusterFactory.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
```

Comment:
pragma once is non-standard

Reply:
Yes but supported by nearly everything: https://en.wikipedia.org/wiki/Pragma_once#Portability

Reply:
@uphoffc, @krenzland, Shall I leave `#pragma once` or replace it with the guards? 

I personally agree with @krenzland, this pragma is supported by almost all compilers and there will be no issues with it.

Reply:
I don't mind, that's for you guys to decide.